### ISSUE TITLE
Training Energy + Forces

### ISSUE THREAD

**Issue by ht141379** [atomistic-machine-learning/schnetpack#223]
Hey I was running through the tutorials in the manual and noticed that in the QM9 tutorial we have the line:
`loss = trn.build_mse_loss([QM9.U0])`
To define the loss function for the trainer. However when looking at the ethanol tutorial there isn't a line. In the Ethanol_script.py in this github it does not run due to mse_loss not being part of the package. So In my attempts I defined my loss as:
`loss = trn.build_mse_loss(properties)`
And defined the mse_loss to include both forces and energy and the rho potential. I was wondering if this is the correct way to do this. 

Additionally wondered if you included a script to plot the predicted values vs the real values to a graph.

**Comment by Stefaanhess** [comment]
Hey @ht141379 ,
you are right. Some refactorinbg had gone wrong in the script. I fixed it in #224 . The easiest way for simple MSE loss is to use the ``build_mse_loss`` and define the tradeoff factors.
Could you explain a bit more, what you mean with the plot for predicted vs. real values?

**Comment by ht141379** [comment]
Hey thanks for that, just wanted some clarification! 
Regarding the second point I was referring to when testing a trained model. I found that in many papers it's a good way to visualize how well your model is preforming by plotting a graph of predicted property vs real property value and we aim for an x=y graph. Essentially plotting the target property value vs the property value predicted by the trained model. If that makes sense?

**Comment by Stefaanhess** [comment]
Hey @ht141379 ,
sorry for the late reply! Unfortunately we do not have a plotting funcion for that, but it should be easy to do it yourself. You can just compute the properties for the test set and plot them depending on the target values

**Comment by ht141379** [comment]
Okay thanks for the info! If you're not too busy I had some other questions!

Thank you

Hayden

**Comment by Stefaanhess** [comment]
Sure, I will try to answer your questions
